src/main/java/nl/novi/techiteasy1121/controllers/CIModuleController.java

```diff
@@ -0,0 +1,58 @@
+package nl.novi.techiteasy1121.controllers;
```

Comment:
Zorg ervoor dat je consistent bent in de naamgeving van je pakketten en klassen. Gebruik camelCase voor pakketten zoals 'nl.novi.techiteasy1121.dtos' in plaats van 'nl.novi.techiteasy1121.Dtos'.